presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -60,7 +60,7 @@
     private boolean legacyOrderBy;
     private boolean legacyMapSubscript;
     private boolean optimizeMixedDistinctAggregations;
-    private boolean distributedSort = false;
+    private boolean distributedSort = true;
```

Comment:
This commit does not look like something we want to land in code base.
Please pass appropriate session property to the runner, have a look for something similar here: https://github.com/prestodb/presto/pull/7300/commits/df87b328c9e0b73e36a5a5b19834285f83c42513.